src/pybind/mgr/dashboard/run-frontend-e2e-tests.sh

```diff
@@ -55,24 +55,29 @@ if [ "$BASE_URL" == "" ]; then
     ./bin/ceph dashboard set-rgw-api-secret-key $(./bin/radosgw-admin user info --uid=dev | jq -r .keys[0].secret_key)
     # Set SSL verify to False
     ./bin/ceph dashboard set-rgw-api-ssl-verify False
+    # Enable Prometheus module
+    ./bin/ceph mgr module enable prometheus
+    # Set Alertmanager API Host
+    ./bin/ceph dashboard set-alertmanager-api-host 'http://localhost:9093'
+    # Set Prometheus server
+    ./bin/ceph dashboard set-prometheus-api-host 'http://localhost:9090'
 
-    BASE_URL=$(./bin/ceph mgr services | jq -r .dashboard)
+    BASE_URL=$(./bin/ceph mgr services | jq .dashboard)
```

Comment:
All changes in this file beginning with line 65 seem unrelated  to the actual issue, I'm also not sure if all these changes are helpful for Jekins. If they are needed could you please put them into a different commit?

Reply:
That was actually a mistake. Only the extra lines on 58-63 lines were meant to change. Thanks for catching, I'll change it back